docs/program/extend/modular-resources/examples/add-tflite-module.md

```diff
@@ -0,0 +1,325 @@
+---
+title: "Add a TensorFlow Lite Modular Service"
+linkTitle: "Add a TensorFlow Lite Modular Service"
+weight: 70
+type: "docs"
+description: "Add a MLModel modular-resource-based service which uses TensorFlow Lite to classify audio samples."
+tags: ["ml", "model training", "services"]
+# SMEs: Andrew Morrow
+---
+
+Viam provides an example [modular resource](extend/modular-resources) written in C++ that extends the [ML model](/services/ml/) service to run any TensorFlow Lite model.
+The example includes an inference client program as well, which generates audio samples and uses the modular resource to classify the audio samples based on a pre-trained model.
+
+This example is intended to demonstrate the design, implementation, and usage of a custom module to serve as a guide to help you write your own.
+This code is for instructional purposes only, and is not intended for production use.
```

Comment:
What does this actually mean? Is this a licensing thing, and people should know that they're not allowed to copy the code for production use? Or is it a functionality thing, and maybe the code isn't as robust/efficient/mature as one would want for production use?

Reply:
Cool. I'm thinking this could be made more clear to users here.